### ISSUE TITLE
Use English as base communication language

### ISSUE THREAD

**Issue by joseigor** [evcc-io/eebus#6]
Hi Guys, 

First of all, you are doing a really good job here. But I would to know if there is the possibility to adopt English as the main base language for discussion on PR and Issues. The reason is that for those who do not speak German is very hard to follow the discussions.

The same for the Documentation and website (for those I can even contribute in my spare time).

BRs, 

Jose

**Comment by andig** [comment]
@joseigor we can't dictate what language people should use. Our main user base is in ASG and remaining open is key. You can probably have good success by replying in english in the threads of interest. Especially for development-related topics (issues, prs) that should work.

**Comment by andig** [comment]
> The same for the Documentation and website (for those I can even contribute in my spare time).

Would be great if you want to contribute. Would require both initial update as well as ongoing maintenance though. We could take care of providing required infrastructure. Seems your request is not specific to eebus though but rather targets evcc in total?

/cc @naltatis

**Comment by andig** [comment]
Duplicate of https://github.com/evcc-io/evcc/issues/4096
